docs/design_docs.md

```diff
@@ -0,0 +1,5 @@
+
+Design Docs:
```

Comment:
nit: to be consistent with the rest of the docs, please make this an actual header with '#'